justfile

```
@@ -0,0 +1,7 @@
+repository := "mesosphere"
```

We need to specify the target registry, otherwise `oras push` will fail
```suggestion
reqistry := "registry-1.docker.io"
repository := "mesosphere/kommander-applications"
```